ingest/settings.py

```diff
@@ -0,0 +1,64 @@
+# File is responsible for defining globals and initializing them
+from mongo_connection import MongoConnection
+from bson.objectid import ObjectId
+
+MONGO_CONNECTION = MongoConnection()
+
+
+def init(study_id, study_file_id):
+    global study
+    study = Study(study_id)
+    global study_file
+    study_file = StudyFile(study_file_id)
+
+
+def get_study():
+    global study
+    return study
+
+
+def get_study_file():
+    global study_file
+    return study_file
+
+
+class Study:
+    STUDY_ACCESSION: str = None
+
+    def __init__(self, study_id):
+        Study.update_study_accession(study_id)
+
+    @classmethod
+    def update_file_type(cls, file_type):
+        cls.FILE_TYPE = file_type
+
+    @classmethod
+    def update_study_accession(cls, study_id):
+        study_accession = list(
+            MONGO_CONNECTION._client["study_accessions"].find(
+                {"study_id": ObjectId(study_id)}, {"accession": 1, "_id": 0}
+            )
+        ).pop()
+        cls.STUDY_ACCESSION = study_accession["accession"]
```

Comment:
It seems faster, simpler, and more robust to pass `study_accession` as an input argument to `ingest_pipeline.py` from Rails.